packages/flutter/test/widgets/gesture_disambiguation.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter/gestures.dart';
+
+void main() {
+  testWidgets('onTap detection with canceled pointer and a drag listener', (WidgetTester tester) async {
+    int detector1TapCount = 0;
+    int detector2TapCount = 0;
+
+    final Widget widget = new GestureDetector(
+        child: new Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: <Widget>[
+              new GestureDetector(
+                  onTap: () => detector1TapCount++,
```

Comment:
various nits:

`=>` returns the value of the expression. However, onTap is defined to return void. Technically Dart is ok with this but from a stylistic perspective we prefer to make it clear by not returning anything.

The `++` postincrement operator is generally something we avoid unless it's really what you want, because conceptually it requires storing a value temporarily. Personally I prefer `+= 1` but the preincrement operator is used by others on the team if you prefer.

In conclusion, you want:

`onTap: () { detector1TapCount += 1; },`

Reply:
done